src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -616,6 +616,16 @@ public T execute(final String merchantAccount, final PaymentData paymentData, fi
         }
     }
 
+    private final Map<String, Object> propertiesToMapWithPropertyFiltering(final Iterable<PluginProperty> properties, TenantContext context) {
+        final Map<String, Object> map = PluginProperties.toMap(properties);
+        for (String sensitiveKey : getConfigProperties(context).getSensitivePropertyKeys()) {
+            if (map.containsKey(sensitiveKey)) {
```

Comment:
Nit: you don't even need this - you could always call `remove`.

Reply:
Done in a138ce2